.pre-commit-config.yaml

```diff
@@ -120,6 +32,13 @@ repos:
             )$
 
 -   repo: local
+
+    hooks:
+    -   id: pylint
+        name: pylint
+        language: system
+        exclude: *exclude_files
+
     hooks:
```

Comment:
@sphuber I didn't review this PR but I think you might have just disabled pylint there...

Reply:
The configured hook doesn't actually work - you'll need to add
```
entry: pylint
```

Reply:
Thanks for the heads-up @ltalirz ! That is indeed a required key but normally that should raise a an error when the file is parsed
```
An error has occurred: InvalidConfigError: 
==> File .pre-commit-config.yaml
==> At Config()
==> At key: repos
==> At Repository(repo='local')
==> At key: hooks
==> At Hook(id='pylint')
=====> Missing required key: entry
```
The bigger problem is that I accidentally defined the `hooks` key twice. The first one, containing the `pylint` config simply gets overridden, which is why `pylint` is never actually run. I will create a PR to fix it.

Reply:
See PR #4258

Reply:
> The bigger problem is that I accidentally defined the hooks key twice.

Right. That's what I wrote above, no? ;-)

Reply:
You did, but not in the original post. I read it on my email, which only ever send the first version. So you edit didn't show up Guess I could have noticed when responding here though :)